### PR TITLE
chore(translation): update with crowdin

### PR DESCRIPTION
**Pull request by AugustinMauroy**

### CODE REVIEW

File: TRANSLATION.md [hunk 1]
```diff
@@ -1,76 +1,47 @@
...
+- dateFormat: The date format. It must be a valid [moment.js format](https://momentjs.com/docs/#/displaying/format/).
+- hrefLang: The language code in [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) format.
+- enabled: If the language is enabled or not.
 
-## Localization groups
+## How to become Proofreader
 
-An existing localization group is not required to start translation. You can contribute on Crowdin without it. Think about groups like a basement to communicate with other translators of your locale.
+- Have translated at least 1000 strings
+- Have 200 strings approved
```
Comment (ovflowd) [inline TRANSLATION.md]: From where are these requirements coming?

File: TRANSLATION.md [hunk 2]
```diff
@@ -1,76 +1,47 @@
...
 
-Contact your appropriate localization group, and discuss with them the best possible way to contribute. A list of the localization groups can be found below.
+To become a proofreader, you need to contact the [Node.js Crowdin Manager](https://crowdin.com/profile/ovflowd) and ask to be promoted. The manager will check your request and promote you if you are eligible.
```
Comment (ovflowd) [inline TRANSLATION.md]: You can say to reach out to any Crowdin Manager. The list is available on the project page.

**Review by manishprivet — APPROVED** [review]
LGTM 🚀

File: TRANSLATION.md (hunk 2 above)
Comment (anonrig) [inline TRANSLATION.md]: There are in total of 4 Crowdin Managers. Should we mention all of them?

File: TRANSLATION.md [hunk 3]
```diff
@@ -1,76 +1,47 @@
...
-- [`nodejs-pt`](https://github.com/nodejs/nodejs-pt) Portuguese Community
-- [`nodejs-ro`](https://github.com/nodejs/nodejs-ro) Romanian Community
-- [`nodejs-ru`](https://github.com/nodejs/nodejs-ru) Russian Community
-- [`nodejs-sv`](https://github.com/nodejs/nodejs-sv) Swedish Community
-- [`nodejs-ta`](https://github.com/nodejs/nodejs-ta) Tamil Community
-- [`nodejs-tr`](https://github.com/nodejs/nodejs-tr) Turkish Community
-- [`nodejs-zh-TW`](https://github.com/nodejs/nodejs-zh-TW) Taiwanese Community
-- [`nodejs-uk`](https://github.com/nodejs/nodejs-uk) Ukrainian Community
-- [`nodejs-vi`](https://github.com/nodejs/nodejs-vi) Vietnamese Community
+## For old translators
```
Comment (anonrig) [inline TRANSLATION.md]:
```suggestion
### For old translators
```

File: TRANSLATION.md (hunk 2 above)
Comment (AugustinMauroy) [reply TRANSLATION.md]: good question.
For Joe and Rich I don't they have time to manage Crowdin.

Comment (anonrig) [reply TRANSLATION.md]: I think we should make all of these information public, and we should not forget that: there is a cost of maintaining the correctness of these lines in the long term.

File: TRANSLATION.md [hunk 4]
```diff
@@ -1,76 +1,47 @@
...
-Node.js is a global platform and so this site has many translations. The translation of the site into
-languages other than English is handled by the localization working group of the language in question. If you
-would like to contribute to the translation of nodejs.org, please refer to the following process:
+We use crowdin to manage translations. You can join the Node.js project on [Crowdin](https://crowdin.com/project/nodejs-website).
 
-> Since Apr, 1 2020 translation process moved to [Crowdin](https://crowdin.com/project/nodejs-website)
+## How to translate
 
-## Get started
+1. Join the [Crowdin](https://crowdin.com/project/nodejs-website)
```
Comment (ovflowd) [inline TRANSLATION.md]: "Request to join" instead of Join

File: TRANSLATION.md [hunk 5]
```diff
@@ -1,76 +1,47 @@
...
+  "name": "French",
+  "langDir": "ltr",
+  "dateFormat": "DD.MM.YYYY",
+  "hrefLang": "fr",
+  "enabled": true
+}
+```
 
-Please create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo. Crowdin managers team would be happy to add new languages.
+- code: The language code. It must be the same as the folder name.
```
Comment (ovflowd) [inline TRANSLATION.md]: Could you transform this into a Markdown table? https://www.tablesgenerator.com/markdown_tables

File: TRANSLATION.md [hunk 6]
```diff
@@ -1,76 +1,47 @@
...
-2. Find at least one more translator for your locale
-3. Create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo requesting the creation of a group for your locale
-
-### Group for my locale is archived
-
-If you find the group for your locale is archived:
-
-1. Try to contact members of the group by creating a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo. Include a mention of the group so members get notified of the issue.
-2. If there is no response from members in 7 days and if you have already done 1000 strings or more on Crowdin for your locale, open an issue in <https://github.com/nodejs/admin> requesting the repository be unarchived.
+Our I18n sub team was disbanded.
```
Comment (ovflowd) [inline TRANSLATION.md]: No need to comment this.

Comment (AugustinMauroy) [reply TRANSLATION.md]: Do you think we should remove the 'For old translators' part?

File: TRANSLATION.md (hunk 5 above)
Comment (AugustinMauroy) [reply TRANSLATION.md]: I don't saw the goal of table ?

Comment (ovflowd) [reply TRANSLATION.md]: The table to be formatted explaining each field of the JSON object.

`Field Name | Description | Examples`

File: TRANSLATION.md (hunk 2 above)
Comment (ovflowd) [reply TRANSLATION.md]: Yes, I don't want to have my name published here.

File: TRANSLATION.md (hunk 6 above)
Comment (ovflowd) [reply TRANSLATION.md]: Yes

File: TRANSLATION.md (hunk 1 above)
Comment (AugustinMauroy) [reply TRANSLATION.md]: I couldn't find a document referring to this so I proposed something.

Comment (ovflowd) [reply TRANSLATION.md]: This sounds random. Let's remove the proofreader part for now :)

File: TRANSLATION.md (hunk 2 above)
Comment (AugustinMauroy) [reply TRANSLATION.md]: Ok i will change that !

File: TRANSLATION.md (hunk 5 above)
Comment (AugustinMauroy) [reply TRANSLATION.md]: OK good idea

File: TRANSLATION.md (hunk 1 above)
Comment (AugustinMauroy) [reply TRANSLATION.md]: Shure ?

**Review by shanpriyan — APPROVED** [review]
LGTM. Thanks!

File: TRANSLATION.md [hunk 7]
```diff
@@ -1,76 +1,25 @@
 # Node.js Website Translation Policy
 
-Node.js is a global platform and so this site has many translations. The translation of the site into
```
Comment (ovflowd) [inline TRANSLATION.md:3]: I think the first part of "Node.js is a global platform and so this site has many translations. The translation of the site into languages other than English is handled by" can be kept. And just replace the part of translation groups to Crowdin

File: TRANSLATION.md [hunk 8]
```diff
@@ -1,76 +1,25 @@
...
 
-1. Open [nodejs-website](https://crowdin.com/project/nodejs-website) Crowdin project
-2. Find your locale and start translation. Find more details in [guide for volunteer translators](https://support.crowdin.com/for-volunteer-translators/)
+## How to add a new language
 
-All translated and approved content will be pushed to this repo automatically. You don't need to create any PRs with translation. Just keep localization process on Crowdin.
+Go on `/i18n/config.json` and add the new language to the `locales` array.
 
-Original source can be found in [/locale/en](https://github.com/nodejs/nodejs.org/tree/main/locale/en). If you find any problem with original source, please create a PR with changes directly to `/locale/en`. Crowdin automatically pull all updates within 24 hours.
+Follow the table below to fill the array:
```
Comment (ovflowd) [inline TRANSLATION.md]: Why did you remove the JSON snippet? What I was asking was to both keep the JSON snippet, and add the table as a reference of what each field is 😅

Comment (AugustinMauroy) [reply TRANSLATION.md]: Oups 😓

File: TRANSLATION.md [hunk 9]
```diff
@@ -12,7 +12,19 @@ We use Crowdin to manage translations. You can join the Node.js project on [Crow
 
 Go on `/i18n/config.json` and add the new language to the `locales` array.
 
-Follow the table below to fill the array:
+Fill the language object with the following fields:
+
+```js
```
Comment (nschonni) [inline TRANSLATION.md]:
```suggestion
```json
```

**Review by ovflowd — APPROVED** [review]
LGTM! Let's get this merged :)
